examples/streaming/kafka-hbase-pipeline/src/main/scala/org/apache/gearpump/streaming/examples/pipeline/PipeLineDSL.scala

```diff
@@ -80,12 +81,17 @@ object PipeLineDSL extends App with ArgumentsParser {
     System.setProperty(Constants.GEARPUMP_CUSTOM_CONFIG_FILE, pipeLinePath)
 
     val app = StreamApp("PipeLineDSL", context, appConfig)
-    val producer = app.readFromKafka(kafkaConfig, msg => {
-      val jsonData = msg.msg.asInstanceOf[String]
-      val envelope = read[Envelope](jsonData)
+    val producer = KafkaDSLUtil.createStream[String](app, kafkaConfig, 1, "time-replayable-producer").map{ message =>
```

Comment:
I think KafkaDSLUtil is OK for both Java and Scala. For Scala only, whether using implicit conversion or not depends on whether we want consistency between Java API and Scala API.